### PR TITLE
Triangle

### PR DESCRIPTION
**Pull request by axtens**

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
The PR title is not shedding much light on what this PR does. Could you expand a bit on what it does?

**Comment by axtens** [comment]
Oops. It's supposed to be the exercise for the triangle task. However it's not complete, suggesting that I've PRed too far up the tree.

**Comment by axtens** [comment]
So if you're able, please delete it

**Comment by ErikSchierboom** [comment]
PRs can't be deleted, sorry

**Comment by kotp** [comment]
Probably set it to draft, you can always force push to this branch and adjust the title and then it will be as it should be.  No need to delete.

File: exercises/practice/armstrong-numbers/armstrong-numbers-tests.8th [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+\ "Single digit numbers are Armstrong numbers"    ( 5 armstrong?  ) test_true
+\ "There are no 2 digit Armstrong Numbers"    ( 10 armstrong?  ) test_false
+\ "Three digit number that is not an Armstrong number"    ( 100 armstrong?  ) test_false
+\ "Three digit number that is an Armstrong number"    ( 153 armstrong?  ) test_true
+\ "Four digit number that is an Armstrong number"    ( 9474 armstrong?  ) test_true
+\ "Four digit number that is not an Armstrong number"    ( 9475 armstrong?  ) test_false
+\ "Seven digit number that is not an Armstrong number"    ( 9926314 armstrong?  ) test_false
+\ "Seven digit number that is an Armstrong number"    ( 9926315 armstrong?  ) test_true
 
 bye
```
Comment (kotp) [inline exercises/practice/armstrong-numbers/armstrong-numbers-tests.8th:11]: Missing last line, by definition.  This happens 18 times for this patch.

Each file that consists of lines, should have an EOL character for the last intended line.

**Review by kotp — CHANGES_REQUESTED** [review]
Other than consistency for line endings, looks good to me.

**Comment by axtens** [comment]
EOL as in control-Z or just a CRLF at the end of the previous line?

**Comment by kotp** [comment]
> EOL as in control-Z or just a CRLF at the end of the previous line?

I believe control-z would give you end of data, not end of line.  The files are using `\n` as shown, `60 60 60 0A` as the last characters for the README while `practice/triangle/triangle_tests.8th` has `62 79 65` as the last, missing the end of line character.  The `0A` hex is `LF`.

I believe you can set your git configuration so that this remains consistent for your changes.

It might be worth setting a `.gitattributes` file in the project so everyone gets the benefit of the settings regarding end of line consistency, as well.

**Pull request merged**